java/org/apache/tomcat/util/http/fileupload/util/Streams.java

```diff
@@ -106,12 +106,12 @@ public static long copy(final InputStream inputStream,
             }
             if (out != null) {
                 if (closeOutputStream) {
-                    out.close();
+                    out.close(); //fixme will closing always with try
```

Comment:
This package(`org.apache.tomcat.util.http.fileupload`) is internal fork of Commons File Upload.  Please do not modify any of the code in this.